hw/hw13/trigrams.py

```diff
@@ -0,0 +1,87 @@
+import string
+import random
+
+
+def strip_cover(infile):
+    #  strip header and footer, then store the whole book as a string
+    wholebook = infile.read(-1)
+    start_string = 'SIR ARTHUR CONAN DOYLE'
+    end_string = 'End of the Project Gutenberg EBook'
```

Comment:
This is good, but it only works for books written by 'SIR ARTHUR CONAN DOYLE' and end with the same end string.